src/java.base/share/classes/javax/crypto/NullCipher.java

```diff
@@ -30,7 +30,9 @@
  * "identity cipher" -- one that does not transform the plain text.  As
  * a consequence, the ciphertext is identical to the plaintext.  All
  * initialization methods do nothing, while the blocksize is set to 1
- * byte.
+ * byte.  Unlike other ciphers, the {@code NullCipher} will not throw an
+ * {@code IllegalStateException} when calling {@code Cipher} methods
+ * in an incorrect state.
```

Comment:
Suggest rewording last part as "... when {@code Cipher} methods are called in an incorrect state." I also think we should say a NullCipher has no state. So how about:

"Unlike other ciphers, the {@code NullCipher} has no state, and will not throw an {@code IllegalStateException} when {@code Cipher} methods are called in an incorrect state."

Reply:
Well, if it has no state it cannot be in an incorrect state so the " in an incorrect state" part can be omitted.

Reply:
Good point!